### DISCUSSION TITLE
Extension does not connect to backend.

### DISCUSSION THREAD

**Discussion by ThisGirlLise** [silveredgold/beta-protection#8] [Q&A] [answered]
I started the beta protection backend. but the extension doesn't want to connect.
even updating it in the settings, so that it connects to the local backend doesn't solve this?

I get no errors or anything.
the webserver of the backend works as it is supposed to?

update: when IPv6 enabled the backend defaults to the IPv6 version. you will have to manually add the http://localhost/2382

**Comment by silveredgold** [comment]
Interesting point about IPv6 messing with it. So it works fine when using `http://localhost:2382`?

**Reply by ThisGirlLise** [reply]
correct.
Leaving the backend address empty it can't connect, but indeed populating it with `http://localhost:2382` fixes that.
maybe populate the field with this address and make it possible to change it?

**Reply by silveredgold** [reply]
Yeah the backend address being empty is not intended behaviour, seems to be a regression as someone else had that yesterday. I'll look into it.

**Comment by silveredgold** [comment, marked as answer]
The empty backend address should be resolved as of v0.1.5 and it will default to localhost instead, which should fix this issue!

**Reply by boi96** [reply]
i have the local host 2382 but it still does not connect to backend

**Comment by abbypatnel** [comment]
Hi, this is different but I get a message saying CRX REQUIRED PROOF MISSING and I was wondering if that meant anything. I can't download the .crx file

**Reply by silveredgold** [reply]
That's covered in the Installation guide [here](https://silveredgold.github.io/beta-protection/guide/installation.html#download)

**Comment by ThisGirlLise** [comment]
Try the fix or load the unpacked extension.
it’s a problem with the authorisation of the crx file in chrome

 

Groetjes,

 

Lise

(Zij/haar)

📧: ***@***.*** ***@***.***> 

 

Van: abbypatnel ***@***.***> 
Verzonden: maandag 22 augustus 2022 08:43
Aan: silveredgold/beta-protection ***@***.***>
CC: ThisGirlLise ***@***.***>; Author ***@***.***>
Onderwerp: Re: [silveredgold/beta-protection] Extension does not connect to backend. (Discussion #8)

 

Hi, this is different but I get a message saying CRX REQUIRED PROOF MISSING and I was wondering if that meant anything. I can't download the .crx file
